### PR TITLE
Fix meteor istest for packages 6331

### PR DESCRIPTION
**Pull request by tmeasday**
For #6331. As discussed.

### CODE REVIEW

File: packages/meteor/test_environment.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+Meteor.isTest = !!TEST_METADATA.isTest;
+Meteor.isAppTest = !!TEST_METADATA.isAppTest;
+
+
+if (Meteor.isClient && (Meteor.isTest || Meteor.isAppTest)) {
+  Meteor.startup(function() {
+    var testDriverPackage = Package[TEST_METADATA.driverPackage];
+    if (!testDriverPackage) {
+      throw new Error("Can't find test driver package: " + TEST_METADATA.driverPackage + "}");
```
Comment (benjamn) [inline packages/meteor/test_environment.js]: What's that `+ "}"` doing there?

File: tools/runners/run-app.js [hunk 2]
```diff
@@ -193,6 +194,11 @@ _.extend(AppProcess.prototype, {
     } else {
       delete env.METEOR_SETTINGS;
     }
+    if (self.testMetadata) {
+      env.TEST_METADATA = JSON.stringify(self.testMetadata);
+    } else {
+     delete env.TEST_METADATA;
```
Comment (benjamn) [inline tools/runners/run-app.js]: indentation

**Comment by benjamn** [comment]
Looks good to me besides those two nits!

File: packages/meteor/test_environment.js (hunk 1 above)
Comment (tmeasday) [inline packages/meteor/test_environment.js]: Good qn! I think it's just a typo from back converting this from ES2015->ES5

**Pull request merged**